bika/lims/browser/analysisrequest/templates/analysisrequest_retract_pdf.pt

```diff
@@ -62,7 +62,8 @@
     labreasons  python:[alllabreas[key] for key in alllabreas.keys() if key != 'checkbox'];
     client      python:ar.getClient();
     rej_widg    python:ar.getRejectionReasons();
-    reasons     python:ar.getRejectionReasons()[0].get('selected',[]) if rej_widg else [];">
+    reasons     python:ar.getRejectionReasons()[0].get('selected',[]) if rej_widg else [];
+    otherreas   python:ar.getRejectionReasons()[0].get('other', '') if rej_widg else '';">
```

Comment:
You could reuse the same code suggested before, so this would look like:
```
reasons     python: view.get_rejection_reasons('selected')
otherreas   python: view.get_rejection_reasons('other')
```